index.php

```diff
@@ -0,0 +1,68 @@
+<?php
+$nazevParty = "Czechitas";
+$poradatel = "Ondra, Petr, Jirka";
+$misto = "Praha";
+$kapacita = 30;
+
+$ucastnici = explode("\n", file_get_contents("ucastnici.txt"));
+
+if (isset($_GET["pridej"])) {
+    $novyUcastnik = $_GET["pridej"];
+    $ucastnici[] = $novyUcastnik;
+
+    file_put_contents("ucastnici.txt", implode("\n", $ucastnici));
+
+    header("Location: /");
+    exit;
+}
+
+$pocetUcastniku = count($ucastnici);
+
+$volnaKapacita = $kapacita - $pocetUcastniku;
+
+if ($pocetUcastniku == 0) {
+    $pocetUcastniku = "Nikdo nepřijde :-(";
+}
+if ($volnaKapacita <= 0) {
+    $volnaKapacita = "Kapacita je vyčerpána";
+}
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Party <?php echo $nazevParty; ?></title>
+</head>
+<body>
+    <h1>Party <?php echo $nazevParty; ?></h1>
+
+    <ul>
+        <li>Pořadatel: <?php echo $poradatel; ?></li>
+        <li>Místo: <?php echo $misto; ?></li>
```

Comment:
Místo je matoucí, protože se to plete s počtem míst, takže bych to přejmenoval třeba na "Lokalitu" nebo tak něco.

Nebo naopak to ostatní přejmenovat na "Celková kapacita:" "Zbývající kapacita" a "Počet přihlášených účastníků", ale nepoužívat "Místo" 2x na různé věci, zbytečně se vytváří zmatení.